user_manual/pim/sync_thunderbird.rst

```diff
@@ -35,11 +35,11 @@ https://cloud.nextcloud.com/remote.php/dav/addressbooks/users/daniel/Thunderbird
 .. image:: ../images/addressbook_name.png
 
 5. Once installed, synchronize your addressbook by clicking "Synchronize" in the top left corner of CardBook.
-You'll see your address book populate from Nextcloud! 
+You'll see your address book populate from Nextcloud!
 
 .. image:: ../images/synchronize_cardbook.png
 
-The rest of the details of dealing with Thunderbird CardBook are left to the reader... 
+The rest of the details of dealing with Thunderbird CardBook are left to the reader...
 
 Calendar
```

Comment:
It would be nice to add that Thunderbird, unlike macOS, evolution or KOrganizer, does not support the automatic discovery of calendars. Hence you will have add each calendar, that you want to synchronise, individually.

Reply:
Something for another PR as this is purely about whitespace changes 😉